stage_tester/st.py

```diff
@@ -148,6 +170,76 @@ def retrieve_cluster_list(organization, address, proxies, auth, verbose):
         print(cluster)
 
 
+def retrieve_results(address, proxies, auth, input_file, verbose):
+    """Retrieve results from the external data pipeline REST API endpoint."""
+    errors={}
+
+    # input file containing list of clusters
+    with open(input_file, "r") as input_file:
+        # iterate over all cluster names
+        for line in input_file:
+            cluster = line.strip()
+
+            if verbose:
+                print("Cluster: ", cluster)
+
+            # construct URL to get report for one specified cluster
+            url = '{}/v1/clusters/{}/report'.format(address, cluster)
```

Comment:
Just a nitpick. I would use f-strings instead of formatted str.

`url = f"{address}/v1/clusters/{cluster}/report"`

Reply:
hmm I saw Python 3.5 somewhere in our tooling, but probably today we can use f-strings

Reply:
fixed